server/src/test/java/org/elasticsearch/action/admin/cluster/node/tasks/TasksIT.java

```diff
@@ -743,23 +736,20 @@ public void testTaskStoringSuccesfulResult() throws Exception {
         TaskInfo taskInfo = events.get(0);
         TaskId taskId = taskInfo.getTaskId();
 
-        GetResponse resultDoc = client()
```

Comment:
I replaced these with the tasks API calls because it gives me a good opportunity to assert that they find the documents while setting the origin.